pagopa/io-app
# feat: [IOBP-619,IOBP-805] Add payment feedback banner feature flag and remote configuration

## ⚠️ This PR depends on https://github.com/pagopa/io-services-metadata/pull/821 ⚠️

## Short description
This PR adds the remote feature flag and remote configurations for the feedback banner that will be displayed on the successful payment screen.

## List of changes proposed in this pull request
- Adapted the `WalletPaymentFeedbackBanner` to show only if the feature flag is enabled and if it is, shows with a remote content inside;
- Added `isPaymentsFeedbackBannerEnabledSelector` to check if the feedback banner should be shown or not;
- Added `paymentsFeedbackBannerConfigSelector` to get the banner remote configurations

## How to test
- Checkout this PR: https://github.com/pagopa/io-dev-api-server/pull/402
- Run the dev-server
- Start a payment from the payments section;
- When you reach the "success" page, if the feature flag is enabled with the min_app_version, you should be able to see the feedback banner as shown in the following screenshot;

## Preview
<img src="https://github.com/user-attachments/assets/ccb7a9db-c066-40a2-b129-7f3d927bc58c" width="300" />
